docs/commands/run.md

```diff
@@ -18,6 +18,15 @@ floyd run [OPTIONS] [COMMAND]
 | command |    | Command to execute when running your project on Floyd. |
 | command |      | Command to execute when running your project on Floyd. |
 
+!!! warning "Encoding issue!"
+    _This is a common issue between **Windows users**_: it could happens that your Terminal is using a different encoding from the one expected from the remote machine. Here are some examples:
```

Comment:
_This is a common issue for **Windows users**_: This occurs when your Windows terminal is using a different encoding from the one expected by the remote machine. Here are some examples: